<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionCriteriaDialog</class>
 <widget class="QDialog" name="ConnectionCriteriaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>461</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connection criteria selector</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../rsc/espina.qrc">
    <normaloff>:/espina/connection.svg</normaloff>:/espina/connection.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1,1,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Add or remove connections from the criteria:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_layout">
     <item>
      <widget class="QToolButton" name="m_add">
       <property name="toolTip">
        <string>Add a connection to current classification</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../rsc/espina.qrc">
         <normaloff>:/espina/add.svg</normaloff>:/espina/add.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_erase">
       <property name="toolTip">
        <string>Remove a connection to current classification</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../rsc/espina.qrc">
         <normaloff>:/espina/remove.svg</normaloff>:/espina/remove.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_clear">
       <property name="toolTip">
        <string>Clear criteria</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../rsc/espina.qrc">
         <normaloff>:/espina/eraser.png</normaloff>:/espina/eraser.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_ambiguous">
     <property name="text">
      <string>Current criteria:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_criteriaLabel">
     <property name="toolTip">
      <string>Current connection criteria</string>
     </property>
     <property name="text">
      <string>None stablished</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_colorGroup">
     <property name="toolTip">
      <string>Representation colors</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 0px;
    margin-top: 1ex; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; 
}</string>
     </property>
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="m_validLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Valid color</string>
          </property>
          <property name="text">
           <string>Valid:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_validColor">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Valid color</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="m_invalidLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Invalid color</string>
          </property>
          <property name="text">
           <string>Invalid:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_invalidColor">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Invalid color</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="m_incompleteLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Incomplete color</string>
          </property>
          <property name="text">
           <string>Incomplete:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_incompleteColor">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Incomplete color</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="m_unconnectedLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Unconnected color</string>
          </property>
          <property name="text">
           <string>Unconnected:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_unconnectedColor">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Unconnected color</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../rsc/espina.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionCriteriaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionCriteriaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
